### DISCUSSION TITLE
Updating to newer version

### DISCUSSION THREAD

**Discussion by Ernestynian** [uxbox/uxbox#1174] [Q&A] [answered]
Hi,
I've been testing Penpot version 1.6.5 self-hosted using the default docker-compose and I already have some data I would like to preserve after moving to the latest release.

Assuming, that I can export the db and re-import it in new postgres container (maybe separate one this time), will the application automatically migrate the database to a newer version, or is there a way for me to do this manually?

**Comment by niwinz** [comment, marked as answer]
Yes, just update to the latest version, and the database will be migrated, is the way we migrate our own instances.
